sorc/IMSaggregate_mod.f90

```diff
@@ -601,6 +639,15 @@ subroutine read_IMS_onto_model_grid(IMS_obs_file, IMS_ind_path, &
            
            error=nf90_get_var(ncid, id_var, IMS_flag)
            call netcdf_err(error, 'error reading IMS nc data' )
+
+           error=nf90_inq_varid(ncid, 'time', id_time)
+           call netcdf_err(error, 'error reading time id' )
+
+           error=nf90_get_var(ncid, id_time, time)
+           call netcdf_err(error, 'error reading time nc data' )
+
+           error=nf90_get_att(ncid, nf90_global, 'time_coverage_end', datestring)
```

Comment:
Do we need lines 649 and 650? `date_from_file` is only used as `ascifile` or else, so it can be blank/undefined, no need to read it from the netCDF input file, right?

Reply:
Yes, I agree. However, if leave a variable as blank/undefined in f90, it may bring some risk. Now I set it as nc_input when read nc file to avoid any risk. What do you think?

Reply:
Thank you, @CoryMartin-NOAA, as this PR is closely associated with ioda-converter Ben is working on, it is good to move forward. Have a great day.